src/AppInstallerCLICore/Commands/SettingsCommand.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "SettingsCommand.h"
+#include "Workflows/WorkflowBase.h"
+#include "Resources.h"
+#include <winget/UserSettings.h>
+
+namespace AppInstaller::CLI
+{
+    using namespace Utility::literals;
+    using namespace AppInstaller::Settings;
+
+    using namespace std::string_view_literals;
+
+    std::vector<Argument> SettingsCommand::GetArguments() const
+    {
+        return {};
+    }
+
+    Resource::LocString SettingsCommand::ShortDescription() const
+    {
+        return { Resource::String::SettingsCommandShortDescription };
+    }
+
+    Resource::LocString SettingsCommand::LongDescription() const
+    {
+        return { Resource::String::SettingsCommandLongDescription };
+    }
+
+    std::string SettingsCommand::HelpLink() const
+    {
+        return "https://aka.ms/winget-settings";
```

Comment:
>https://aka.ms/winget-settings [](start = 16, length = 30)

Did someone create this?

Reply:
I did, is not pointing to docs right now, but will update it once settings.md is on master

---
In reply to: [442413601](https://github.com/microsoft/winget-cli/pull/436#discussion_r442413601) [](ancestors = 442413601)